_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -13336,7 +13336,9 @@
 	},
 /obj/machinery/requests_console{
 	name = "Detective Requests Console";
-	pixel_y = -30
+	pixel_y = -30;
+	department = "Detective";
+	departmentType = 5
```

Comment:
Same as above, if you consider giving the detective a disposal bin, the Cyberiad doesn't have one either.